Adjusting User guide structure (#42)
mckinsey/vizro: vizro-core/docs/pages/user_guides/selectors.md

# How to use selectors

> This guide shows you the differences in selectors for your dashboard. Selectors do not serve a function on their own, but they let you _select_ the input for other models, e.g. the [`Filter`][vizro.models.Filter] or the [`Parameter`][vizro.models.Parameter] model.

Reviewer: Should `_select_` be changed to ``select``?

Author: It was supposed to be a highlight, as in why its a basically called slector. `_abc_` just makes things itallic.

> The [`Filter`][vizro.models.Filter] or the [`Parameter`][vizro.models.Parameter] model accept the `selector` argument, where you can enter either a selecetor model if the automatic default does not serve your needs.

Reviewer: ```suggestion
The [`Filter`][vizro.models.Filter] or the [`Parameter`][vizro.models.Parameter] model accept the `selector` argument, where a selecetor model can be entered if the default does match the requirement.
```
